src/main/java/com/minecolonies/blockout/controls/TextField.java

```diff
@@ -50,7 +50,6 @@ public TextField()
 
     /**
      * Public constructor to instantiate the field with params.
-     * @param params
      */
     public TextField(@NotNull final PaneParams params)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document the parameter(s): params [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)